apps/docs/content/guides/integrations/platform-as-a-service.mdx

```diff
@@ -0,0 +1,393 @@
+
+Get the new API keys by making a [`GET /v1/projects/{ref}/api-keys`](https://api.supabase.com/api/v1#tag/secrets/get/v1projectsrefapi-keys) request.
+
+```sh
+curl 'https://api.supabase.com/v1/projects/{ref}/api-keys?reveal=true' \
+  --header 'Authorization: Bearer YOUR_SECRET_TOKEN'
+```
+
+If the response includes `"publishable"` and `"secret"` keys then you're all set and you should only use those from now on.
+
+Otherwise, enable the new API keys by making two [`POST /v1/projects/{ref}/api-keys`](https://api.supabase.com/api/v1#tag/secrets/post/v1projectsrefapi-keys) requests, one for `publishable` and another for `secret`.
+
+```sh
+curl 'https://api.supabase.com/v1/projects/{ref}/api-keys' \
+  --request POST \
+  --header 'Content-Type: application/json' \
+  --header 'Authorization: Bearer YOUR_SECRET_TOKEN' \
+  --data '{
+  "type": "publishable",
+  "name": "default"
+}'
+```
+
+```sh
+curl 'https://api.supabase.com/v1/projects/{ref}/api-keys?reveal=true' \
+  --request POST \
+  --header 'Content-Type: application/json' \
+  --header 'Authorization: Bearer YOUR_SECRET_TOKEN' \
+  --data '{
+  "type": "secret",
+  "name": "default",
+  "secret_jwt_template": {
+    "role": "service_role"
+  }
+}'
+```
+
+## Changing compute sizes
+
+Management API endpoint: [`PATCH /v1/projects/{ref}/billing/addons`](https://api.supabase.com/api/v1#tag/billing/patch/v1projectsrefbillingaddons)
+
+You can upgrade and downgrade compute sizes by making requests to [`PATCH /v1/projects/{ref}/billing/addons`](https://api.supabase.com/api/v1#tag/billing/patch/v1projectsrefbillingaddons).
+
+```sh
+curl 'https://api.supabase.com/v1/projects/{ref}/billing/addons' \
+  --request PATCH \
+  --header 'Content-Type: application/json' \
+  --header 'Authorization: Bearer YOUR_SECRET_TOKEN' \
+  --data '{
+    "addon_type": "compute_instance",
+    "addon_variant": "ci_small"
+  }'
+```
+
+Once you have access to [Pico instances](#pico-compute-instance), you will no longer be able to upgrade or downgrade to Nano instances.
+
+## Configuration changes
+
+Management API endpoints:
+
+- Auth: [`PATCH /v1/projects/{ref}/config/auth`](https://api.supabase.com/api/v1#tag/auth/patch/v1projectsrefconfigauth)
+- Data API (PostgREST): [`PATCH /v1/projects/{ref}/postgrest`](https://api.supabase.com/api/v1#tag/rest/patch/v1projectsrefpostgrest)
+- Edge Functions:
+  - [`PATCH /v1/projects/{ref}/functions/{function_slug}`](https://api.supabase.com/api/v1#tag/edge-functions/patch/v1projectsreffunctionsfunction_slug)
+  - [`PUT /v1/projects/{ref}/functions`](https://api.supabase.com/api/v1#tag/edge-functions/put/v1projectsreffunctions)
+- Storage: [`PATCH /v1/projects/{ref}/config/storage`](https://api.supabase.com/api/v1#tag/storage/patch/v1projectsrefconfigstorage)
+- Realtime: [`PATCH /v0/projects/{ref}/config/realtime`](https://api.supabase.com/api/v0#tag/projects-config/patch/v0projectsrefconfigrealtime) (moving to v1 soon)
```

Comment:
Remember to swap out once this is up on v1. It was just merged: https://github.com/supabase/infrastructure/pull/27788